Mathlib/Data/PFunctor/Univariate/Basic.lean

```diff
@@ -72,35 +70,34 @@ protected theorem id_map : ∀ x : P α, P.map id x = x := fun ⟨_, _⟩ => rfl
 protected theorem map_map (f : α → β) (g : β → γ) :
     ∀ x : P α, P.map g (P.map f x) = P.map (g ∘ f) x := fun ⟨_, _⟩ => rfl
 
-instance : LawfulFunctor.{v, max u v} P.Obj where
+instance : LawfulFunctor (Obj.{uA, uB, v} P) where
```

Comment:
Like I said earlier, can you move `v` earlier in the `universe` line so that this can be spelt as follows?
```suggestion
instance : LawfulFunctor (Obj.{v} P) where
```
The other two universes are completely determined by P, so should come last, where they are effectively implicit.